### ISSUE TITLE
Accessibility statement

### ISSUE THREAD

**Issue by penx** [govuk-react/govuk-react#465]
Produce ACCESSIBILITY.md in the root of the project, or similar, covering:

- where we are with accessibility
- aim to track govuk-frontend 100%
- server side vs. client side vs. universal rendering
- W3C spec compliance vs. real world issues - client side routing, ARIA on universal
- refer to govuk standards, guidelines, requirements - and any statements on when they are must haves vs. should haves
  - https://www.gov.uk/service-manual/technology/using-progressive-enhancement
  - https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps
- form submission on universal applications
- related issues
- links to examples
- reach router
- next.js
- https://reactjs.org/docs/accessibility.html
- encourage contributions in the form of issues and PRs
- some components are designed to be client side/universal e.g. LoadingBox

Related: https://github.com/UKHomeOffice/govuk-react/issues/460

**Comment by penx** [comment]
Cheers @Joe-Brady 

My understanding is that if we're tracking govuk-frontend's markup and styles, and doing server side rendering only (which I think you are), we should be able to meet the same accessibility standards as govuk-frontend.

At the moment there are probably places where we're not using exactly the same markup and may be missing aria tags. I am guessing this is the majority of the feedback, looking forward to seeing it.

Related 
https://github.com/UKHomeOffice/govuk-react/issues/450
